src/pages/CatIndex.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component } from 'react'
+// import { Card, CardTitle, Col } from 'reactstrap'
+
 
 class CatIndex extends Component {
     render(){
+        console.log(this.props.cats)
         return(
             <>
                 <h1>Here are all the cats!</h1>
+                { this.props.cats.map((cat, index) => {
+                    return (
+                        //<p key={index}>{ cat.name}</p>
```

Comment:
If you are not using this first p tag, go ahead and delete it

Reply:
Copy!